### PR TITLE
added entry in the glossary for Encounter

### PR DESCRIPTION
**Pull request by bleehu**
modified:   03_Glossary_of_Terms.txt

### CODE REVIEW

File: 03_Glossary_of_Terms.txt [hunk 1]
```diff
@@ -53,6 +53,15 @@ limited by turns with certain amounts of Actions. See Basic Rules.txt section
 == D10 ==
     A 10-sided Die. See "The Primary Game Mechanic" in the Basic Rules.
 
+== Encounter ==
+    An encounter is an arbitrary unit of measure of role playing gameplay. An
+encounter is one puzzle, one battle, or one plot-progressing social engagement
+with one or more NPCs. 
+
+    When measuring a battle encounter, the encounter is the time from making 
+first stealth attack or rolling iniative to the time that play is no longer done
```
Comment (rosebrallier) [inline 03_Glossary_of_Terms.txt]: sp: initiative

File: 03_Glossary_of_Terms.txt [hunk 2]
```diff
@@ -53,6 +53,15 @@ limited by turns with certain amounts of Actions. See Basic Rules.txt section
...
+in iniative order.
```
Comment (rosebrallier) [inline 03_Glossary_of_Terms.txt]: Personally, I'd straight up nix "done" in this sentence. Reads more fluidly and imo doesn't detract from the meaning.
sp: initiative

**Review by rosebrallier — APPROVED** [review]
Only non-blocking comments. Shipit!

File: 03_Glossary_of_Terms.txt (hunk 1 above)
Comment (bleehu) [reply 03_Glossary_of_Terms.txt]: gadzooks. How do I misspell initiative?

**Pull request merged**